src/api/app/controllers/webui/request_controller.rb

```diff
@@ -334,16 +334,16 @@ def forward_request_to(fwd)
   end
 
   def request_action_attributes(type)
-    {
-      target_project: params[:project],
-      target_package: params[:package],
-      source_project: params[:devel_project],
-      source_package: params[:devel_package] || params[:package],
-      target_repository: params[:repository],
-      person_name: params[:user],
-      group_name: params[:group],
-      role: params[:role],
-      type: type.to_s
-    }
```

Comment:
can you explain why is it better than before?

Reply:
We pass the params of user or group only if they are present. Otherwise it would result into a crashing of the follow up functions which uses `request_action_attributes`